contracts/Governance/GovernorBravoDelegate.sol

```diff
@@ -57,18 +55,29 @@ contract GovernorBravoDelegate is GovernorBravoDelegateStorageV1, GovernorBravoE
         require(msg.sender == admin, "GovernorBravo::initialize: admin only");
         require(timelock_ != address(0), "GovernorBravo::initialize: invalid timelock address");
         require(xvsVault_ != address(0), "GovernorBravo::initialize: invalid xvs address");
-        require(votingPeriod_ >= MIN_VOTING_PERIOD && votingPeriod_ <= MAX_VOTING_PERIOD, "GovernorBravo::initialize: invalid voting period");
-        require(votingDelay_ >= MIN_VOTING_DELAY && votingDelay_ <= MAX_VOTING_DELAY, "GovernorBravo::initialize: invalid voting delay");
-        require(proposalThreshold_ >= MIN_PROPOSAL_THRESHOLD && proposalThreshold_ <= MAX_PROPOSAL_THRESHOLD, "GovernorBravo::initialize: invalid proposal threshold");
         require(guardian_ != address(0), "GovernorBravo::initialize: invalid guardian");
+        require(timelocks.length == uint8(ProposalType.CRITICAL) + 1, "number of timelocks should match number of governance routes");
+        require(proposalConfigs_.length == uint8(ProposalType.CRITICAL) + 1, "number of timelocks should match number of governance routes");
+
 
         timelock = TimelockInterface(timelock_);
         xvsVault = XvsVaultInterface(xvsVault_);
-        votingPeriod = votingPeriod_;
-        votingDelay = votingDelay_;
-        proposalThreshold = proposalThreshold_;
         proposalMaxOperations = 10;
         guardian = guardian_;
+
+        //Set parameters for each Governance Route
+        for(uint8 i = 0; i<proposalConfigs_.length; ++i){
```

Comment:
i think we can use uint256 instead of uint8 and just don't assign i=0 as it just waste of gas.

References 
- [uint256-can-be-cheaper-than-uint8](https://yos.io/2021/05/17/gas-efficient-solidity/#tip-2-uint256-can-be-cheaper-than-uint8)